tests/adafruit_gps_test.py

```diff
@@ -177,6 +177,10 @@ def test_GPS_update_rmc_no_magnetic_variation():
         assert gps.timestamp_utc == exp_time
         assert gps.latitude == pytest.approx(12.57613)
         assert gps.longitude == pytest.approx(1.385391)
+        assert gps.latitude_degrees == 12
+        assert gps.longitude_degrees == 1
+        assert gps.latitude_minutes == 34.5678
+        assert gps.longitude_minutes == 3.12345
```

Comment:
I believe this should be 23.12346.  I'm surprised it passed, if so.

Reply:
Good catch. That's a bug when parsing the NMEA sentence. I'll try to fix it.